$NetBSD: patch-misc_terminfo.src,v 1.3 2020/03/28 07:55:55 wiz Exp $

Many terminal emulators still do not support the "rep" capability, so
avoid it for maximum portability.  Patch from OmniOS, see also
http://invisible-island.net/ncurses/ncurses.faq.html#xterm_generic
or
https://github.com/joyent/pkgsrc/issues/248

--- misc/terminfo.src.orig	2020-01-19 01:09:38.000000000 +0000
+++ misc/terminfo.src
@@ -4513,7 +4513,7 @@ xterm-xfree86|xterm terminal emulator (X
 # This version reflects the current xterm features.
 xterm-new|modern xterm terminal emulator,
 	npc,
-	kcbt=\E[Z, kent=\EOM, use=ecma+index, use=ansi+rep,
+	kcbt=\E[Z, kent=\EOM, use=ecma+index,
 	use=ecma+strikeout, use=xterm+keypad, use=vt420+lrmm,
 	use=xterm+sm+1006, use=xterm+pcfkeys, use=xterm+tmux,
 	use=xterm-basic,
